p2panda-rs/src/schema/system/schema_definition.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use std::collections::BTreeMap;
+
+use crate::schema::{FieldType, Schema, SchemaId, SchemaIdError};
+
+const DESCRIPTION: &str = "Publish data schemas for your application.";
+
+/// Returns the `schema_definition` system schema with a given version.
+pub fn get_schema_definition(version: u8) -> Result<Schema, SchemaIdError> {
```

Comment:
Nice! This could be also an candidate for `lazy_static` (see how we use it here: https://github.com/p2panda/p2panda/blob/main/p2panda-rs/src/cddl/definitions.rs#L193), then it will be assured that this method is only executed once, ever!